Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+elifePipeline {
+    def tag
+    DockerImage image
+
+    node('containers-jenkins-plugin') {
+        stage 'Checkout', {
+            checkout scm
+            def commit = elifeGitRevision()
+            def commitShort = commit.substring(0, 8)
+            tag = "${commitShort}-${params.SCHEMA_VERSION}"
```

Comment:
needs a default or will normally be casted to the string `null`

Reply:
doesn't need to be uppercase?

Reply:
Doesn't have to be, just a Jenkins parameter after all, is there a convention that you follow/prefer e.g. kebab case, etc?
I've set a default value in the Jenkins project itself, but can play it safe and add a default value here again.

Reply:
all lower case, see https://alfred.elifesciences.org/job/prod-journal/build?delay=0sec or https://github.com/elifesciences/pattern-library/blob/develop/Jenkinsfile.prod#L24

The default in Jenkins is in https://alfred.elifesciences.org/job/test-basex-validator-docker/ but that only applies to the mainline build, not to pull requests which are detected and built at https://alfred.elifesciences.org/job/pull-requests-projects/job/basex-validator-docker/view/change-requests/ automatically. Also, prefer values stored in the source to values stored in Jenkins as they are easier to find, modify, etc.

Reply:
Have adjusted, renamed to 'schemaVersion' and cleared the default value in Jenkins, falling back to 'master' in the script.